### DISCUSSION TITLE
WRF Hydro 5.3.0 tar file?

### DISCUSSION THREAD

**Discussion by HathewayWill** [NCAR/wrf_hydro_nwm_public#689] [General]
Good morning,

Saw in the latest GitHub for WRF-ARW it mentioned that hydro is up to v5.3.0

But the tar file on the official wrf hydro repo is 5.2.0. Is there an update coming soon?

**Comment by scrasmussen** [comment]
Hi, we have a branch that has [version 5.3.0 ](https://github.com/NCAR/wrf_hydro_nwm_public/tree/v5.3.0). I'm not sure when we will bundle it for the official release but hopefully that is a bit helpful. I make a note to find an answer to when the update is coming and get back to you.

**Reply by HathewayWill** [reply]
Thanks,  since the WRFv4.5 release notes mentioned it as being supported I was confused.
